adaptive_breakpoints/lib/src/adaptive_breakpoints.dart

```diff
@@ -0,0 +1,377 @@
+import 'package:flutter/material.dart';
+
+/// Adaptive Window in Material has five different window sizes. Each one can be
+/// broken into smaller bits.
```

Comment:
I'm not sure I understand what this means

Reply:
Done